### ISSUE TITLE
`gamma` and `beta` in AccumBatchNormalization never updated

### ISSUE THREAD

**Issue by andreped** [andreped/GradientAccumulator#60]
**Describe the bug**
`gamma` and `beta` coefficients in `AccumBatchNormalization` layer are never updated. This is likely the reason why the custom  AccumBN layer does not reach the same results as regular BN in Keras.

A fix likely also need to work for gradient accumulation.

**To Reproduce**
Use [this](https://github.com/andreped/GradientAccumulator/blob/e6283bf7cdf74316091990bb3ed53601f181e6d6/tests/test_batch_norm.py#L65) test to run benchmarks, before and after adding proper `gamma`/`beta` support.

**Expected behavior**
Adding this should ideally reach _identical_ results to the regular BatchNormalization layer in Keras without introducing gradient accumulation.

**Comment by andreped** [comment]
Nvm, these are trainable variables and should be updated during backprop.

Due to this they will also be updated for every `accum_steps` iteration, similarly as for all other weights.

I just observed that the training losses appear _identical_ between the custom and keras layers. It is only the validation loss that is off. Hence, we are doing something wrong during the testing phase.

**Comment by andreped** [comment]
Fixed in 7ba4ac69570bf42688e440ba4acf065930fca497.

`gamma` and `beta` are updated during back propagation. The issue was related to using the wrong `momentum` (0.9 instead of 0.99 that keras uses)
